src/views/catalog/customize/utils.ts

```diff
@@ -40,7 +40,7 @@ export const overrideVirtualMachineDataVolumeSpec = (
 };
 
 export const setTemplateParameters = (template: V1Template, formData: FormData): V1Template => {
-  template.parameters = template.parameters.map((parameter) => {
+  template.parameters = template?.parameters?.map((parameter) => {
```

Comment:
+1

Reply:
Yep, this change is here to prevent the UI to do 💣  unnecessary